app/models/course.rb

```diff
@@ -9,6 +9,7 @@ class Course < ActiveRecord::Base
   belongs_to :creator, class_name: User.name
   has_many :course_users, inverse_of: :course, dependent: :destroy
   has_many :users, through: :course_users
+  has_many :enrol_requests
```

Comment:
According to their documentation "for belongs_to associations has_many inverse associations are ignored". We can still put inverse_of but it won't be that useful I think.